src/app/+home-page/home-news/home-news.component.html

```diff
@@ -5,16 +5,17 @@
         <img src="assets/images/dspace-logo.png" />
       </div>
       <div>
-        <h1 class="display-3">Welcome to DSpace</h1>
-        <p class="lead">DSpace is an open source software platform that enables organisations to:</p>
+        <h1 class="display-3">Welcome to DSpace 7 Alpha</h1>
+        <p class="lead">DSpace is the world leading open source repository platform that enables organisations to:</p>
       </div>
     </div>
     <ul>
-      <li>capture and describe digital material using a submission workflow module, or a variety of programmatic ingest options
+      <li>easily ingest documents, audio, video, datasets and their corresponding Dublin Core metadata
       </li>
-      <li>distribute an organisation's digital assets over the web through a search and retrieval system
+      <li>open up this content to local and global audiences, thanks to the OAI-PMH interface and Google Scholar optimizations
       </li>
-      <li>preserve digital assets over the long term</li>
+      <li>issue permanent urls and trustworthy identifiers, including optional integrations with handle.net and DataCite DOI</li>
     </ul>
+    <p>Join a vibrant community of DSpace enthusiasts including <A HREF="https://dash.harvard.edu" TARGET="_NEW">Harvard University</A>, <A HREF="https://openknowledge.worldbank.org">The World Bank</A>, <A HREF="https://www.repository.cam.ac.uk/" TARGET="_NEW">Cambridge University</A>, <A HREF="http://apps.who.int/iris/" TARGET="_NEW">The World Health Organization</A>, <A HREF="https://dspace.mit.edu/" TARGET="_NEW">MIT</A> and many more.</p>
```

Comment:
I don't think we should call out specific sites in our homepage news.  If somehow one of these sites were down, provided a less than ideal experience, or (hopefully not) switched to a new platform, that'd reflect badly on DSpace.  Instead, we could link to the registry, or just leave out this paragraph altogether.

Reply:
@tdonohue this was discussed in the marketing group today https://wiki.duraspace.org/pages/viewpage.action?pageId=112528617. Generally, there was agreement with your concerns of sites being down/unavailable.

On the other hand, we didn't find it the dspace registry appropriate to link to, because you don't really see selected repositories there. So instead, we're currently linking to the DSpace Positioning page. This might still change if someone makes a better /other page.